layouts/_default/single.html

```diff
@@ -6,64 +6,55 @@
   <body class="{{ printf "bg-%s" $bg }}">
     {{ partial "nav.html" . }}
 
-    {{ $page := replace (strings.TrimRight "/" (strings.TrimLeft "/" .RelPermalink)) "/" "__" }}
-
-    {{ with index $.Site.Data $page }}
-      {{ range . }}
-        {{ $.Scratch.Set "data" . }}
-
-        {{ if and (not .disabled) (eq .fragment "hero") }}
-          {{ partial "fragments/hero.html" $ }}
-        {{ end }}
+    {{ range .Resources }}
 
+      {{ if and (not .Params.disabled) (eq .Params.fragment "hero") }}
+        {{ partial "fragments/hero.html" (dict "self" . ) }}
+      {{ end }}
+      
+      {{ if and (not .Params.disabled) (eq .Params.fragment "items") }}
+        {{ partial "fragments/items.html" (dict "self" . ) }}
+      {{ end }}
+      
+      {{ if and (not .Params.disabled) (eq .Params.fragment "items") }}
+        {{ partial "fragments/items.html" .Params }}
```

Comment:
This seems left over and shouldn't be necessary as the items.html fragment is already called above.